src/components/RTLLayoutDetector.tsx

```diff
@@ -0,0 +1,14 @@
+import { ReactNode } from 'react';
+import useTranslation from 'next-translate/useTranslation';
+
+interface Props {
+  children: ReactNode | ReactNode[];
+}
+
+const RTLLayoutDetector: React.FC<Props> = ({ children }) => {
+  const { lang } = useTranslation();
+
+  return <main dir={lang === 'ar' ? 'rtl' : 'ltr'}>{children}</main>;
```

Comment:
can we also include:
- Uyghur
- Urdu
- Persian
- Hebrew
- Pashto
- Divehi 

all of these are rtl languages.

Reply:
Got it thanks @naveed-ahmad . As per my discussion with Abdellatif, we will postpone this PR until pre-production release so that all pages, layouts and components are in place so that we can test this feature properly and make sure nothing breaks. but I will include those languages once I am back to this PR. Thanks a lot.